backend/src/main/kotlin/rocks/didit/sefilm/domain/Foretagsbiljett.kt

```diff
@@ -0,0 +1,19 @@
+package rocks.didit.sefilm.domain
+
+import com.fasterxml.jackson.annotation.JsonValue
+
+data class Foretagsbiljett(val value: String) {
+  init {
+    if (value.length != 11) {
+      throw IllegalArgumentException("Företagsbiljett has wrong size. Expected 11, got ${value.length}")
+    }
+    if (value.toLongOrNull() == null) {
+      throw IllegalArgumentException("'$value' is an invalid företagsbiljett")
+    }
+  }
+
+  @JsonValue
```

Comment:
_I don't know if you know this, but this is only so that the value is serialized "primitively". I.e. as simple string instead of as an object._